calcom/cal.com
# fix: Allow passing secret for the webhooks via API

## What does this PR do?

We should be able to add secret key for securing webhooks via API. This PR allows that via POST & PATCH calls.



## Requirement/Documentation

- NA

## Type of change




## How should this be tested?

- Send POST and PATCH API for webhooks and include `secret` as part of the API payload.
- When that webhook is triggered, you should see `X-Cal-Signature-256` as part of the header in the webhook hit received.

## Mandatory Tasks


## Checklist
